config-model-fat/pom.xml

```diff
@@ -28,6 +28,13 @@
       <groupId>com.yahoo.vespa</groupId>
       <artifactId>config-model</artifactId>
       <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <!-- Large, and installed separately as part of Vespa -->
```

Comment:
Is 'installed separately' really relevant? I don't think the .so files inside an embedded jar would have been used anyway since that location is not likely to be part of a library path.

Reply:
Are you sure? I can run the jni code locally on my mac, but haven't checked if there are JNI's in the other jar as well.

Reply:
It will try inside the jar after it has checked what is on the library path.

Reply:
So ... ok then?